tests/dualtor_io/test_normal_op.py

```diff
@@ -24,6 +26,11 @@
 ]
 
 
+@pytest.fixture(autouse=True)
+def common_setup_teardown(validate_active_active_dualtor_setup):    # noqa F811
```

Comment:
The empty fixture looks wired. Can we call `validate_active_active_dualtor_setup` explicitly for each test?